[ACRA 2.10alpha71] - Using WebView from more than one process at once with the same data directory is not supported

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/2866f15f-a86a-4fc3-8ece-107747a50271

```
java.lang.RuntimeException: Unable to create application com.ichi2.anki.AnkiDroidApp: java.lang.RuntimeException: Using WebView from more than one process at once with the same data directory is not supported. https://crbug.com/558377 : Lock owner com.ichi2.anki:acra (pid 11402)
at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5911)
at android.app.ActivityThread.access$1100(ActivityThread.java:200)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1660)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loop(Looper.java:193)
at android.app.ActivityThread.main(ActivityThread.java:6762)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.RuntimeException: Using WebView from more than one process at once with the same data directory is not supported. https://crbug.com/558377 : Lock owner com.ichi2.anki:acra (pid 11402)
at org.chromium.android_webview.AwDataDirLock.a(PG:26)
at kn.g(PG:46)
at kn.a(PG:38)
at kn.e(PG:3)
at com.android.webview.chromium.WebViewChromiumFactoryProvider.getStatics(PG:4)
at android.webkit.WebView.setWebContentsDebuggingEnabled(WebView.java:2118)
at com.ichi2.utils.WebViewDebugging.initializeDebugging(WebViewDebugging.java:3)
at com.ichi2.anki.AnkiDroidApp.onCreate(AnkiDroidApp.java:13)
at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1155)
at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5906)
... 8 more
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.java
@@ -228,8 +228,6 @@ public class AnkiDroidApp extends Application {
             UsageAnalytics.setDryRun(true);
         }
 
-        WebViewDebugging.initializeDebugging(preferences);
-
         setLanguage(preferences.getString(Preferences.LANGUAGE, ""));
         NotificationChannels.setup(getApplicationContext());
 

--- a/AnkiDroid/src/main/java/com/ichi2/utils/WebViewDebugging.java
+++ b/AnkiDroid/src/main/java/com/ichi2/utils/WebViewDebugging.java
@@ -12,6 +12,12 @@ public class WebViewDebugging {
     public static void initializeDebugging(SharedPreferences sharedPrefs) {
         // DEFECT: We might be able to cache this value: check what happens on WebView Renderer crash
         // On your desktop use chrome://inspect to connect to emulator WebViews
+        // Beware: Crash in AnkiDroidApp.onCreate() with:
+        /*
+        java.lang.RuntimeException: Using WebView from more than one process at once with the same data directory
+        is not supported. https://crbug.com/558377 : Lock owner com.ichi2.anki:acra at
+        org.chromium.android_webview.AwDataDirLock.a(PG:26)
+         */
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
             boolean enableDebugging = sharedPrefs.getBoolean("html_javascript_debugging", false);
             WebView.setWebContentsDebuggingEnabled(enableDebugging);

